EDITORS.md

```diff
@@ -254,3 +280,52 @@ To configure indexing, pass a JSON hash as part of the initialization options fo
   }
 }
 ```
+
+# All initialization options
```

Comment:
Since this applies to all editors, let's move it up, right below the `IMPORTANT NOTE` section.